nautobot/extras/tests/test_relationships.py

```diff
@@ -106,6 +106,30 @@ def test_clean_filter_not_valid(self):
         with self.assertRaises(ValidationError):
             m2m.clean()
 
+        m2m = Relationship(
+            name="Another Vlan to Rack",
+            slug="vlan-rack-2",
+            source_type=self.site_ct,
+            source_filter={"site": "not a list"},
+            destination_type=self.rack_ct,
+            type=RelationshipTypeChoices.TYPE_MANY_TO_MANY,
+        )
+
+        with self.assertRaises(ValidationError):
+            m2m.clean()
```

Comment:
Can we expand these checks to verify the contents of the ValidationError? Something like:
```suggestion
        with self.assertRaises(ValidationError) as handler:
            m2m.clean()
        self.assertEqual(handler.exception.message_dict, ...)
```

Reply:
Thanks, I've updated all unit tests for relationship to check the error messages